lib/ui/useable-components/data-view/index.tsx

```
@@ -0,0 +1,43 @@
+import React from 'react';
+import { DataView } from 'primereact/dataview';
```

Biome

[error] Do not shadow the global "DataView" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)